lib/util/fsUtil.js

```diff
@@ -25,4 +25,5 @@ module.exports = {
 
   isMarkdown: filePath => path.extname(filePath) === '.md',
   isHtml: filePath => path.extname(filePath) === '.html',
+  isSourceFile: filePath => module.exports.isMarkdown(filePath) || module.exports.isHtml(filePath),
```

Comment:
We don't usually use `module.exports` when exporting functions.
I think if you change to use function object syntax then you can use `this` to reference the functions defined in the object.
```
isSourceFile: function(filePath) {
  return this.isMarkdown(filePath) || this.isHtml(filePath);
}
```
Or use ES6 getter syntax
```
get isSourceFile() {
  return filePath => this.isMarkdown(filePath) || this.isHtml(filePath);
}
```